pkg/protocol/sybilprotection/sybilprotectionv1/sybilprotection.go

```diff
@@ -226,17 +225,10 @@ func (o *SybilProtection) committeeRoot(targetCommitteeEpoch iotago.EpochIndex)
 		iotago.AccountIDFromBytes,
 	)
 
-	var innerErr error
-	committee.ForEach(func(accountID iotago.AccountID, _ *account.Pool) bool {
-		if err := committeeTree.Add(accountID); err != nil {
-			innerErr = ierrors.Wrapf(err, "failed to add account %s to committee tree", accountID)
-			return false
+	for _, accountID := range committee.IDs() {
+		if err = committeeTree.Add(accountID); err != nil {
```

Comment:
Since we export seat numbers for a snapshot (via `o.performanceTracker.Export`) we should really make sure that they are the same across nodes as otherwise snapshots at the same epoch can't be directly compared (eg via hashes). Would it make sense to commit to the seat here in a `ads.Map`?

Reply:
Damn, this is something I completely missed. There is actually no reason why nodes would be required to assign the same seats - this should be completely subjective. Maybe instead we should just export accounts to the snapshot so that it can be hashed and compared, and then nodes initialize their seats when loading the snapshot? Then each node could assign seats differently without causing much problems.

Reply:
Yep that sounds good. Just need to make sure to document this properly so that it is clear that `Export` and `SaveToDisk` (or whatever we'll call it) are different in this way.

Reply:
I changed it and now we only export Accounts to the snapshot along with the "reused" flag.
When loading the snapshot, we don't maintain seats for past slots as I think this is only required for the current epochs to track seat activity correctly - or maybe I'm missing something? I provided extensive comments describing the reasoning.